posthog/temporal/batch_exports/s3_batch_export.py

```diff
@@ -103,13 +103,16 @@ def get_allowed_template_variables(inputs) -> dict[str, str]:
     }
 
 
-def get_s3_key(inputs) -> str:
+def get_s3_key(inputs, file_number: int | None = None) -> str:
     """Return an S3 key given S3InsertInputs."""
     template_variables = get_allowed_template_variables(inputs)
     key_prefix = inputs.prefix.format(**template_variables)
     file_extension = FILE_FORMAT_EXTENSIONS[inputs.file_format]
 
     base_file_name = f"{inputs.data_interval_start}-{inputs.data_interval_end}"
+    # to maintain backwards compatibility with the old file naming scheme
```

Comment:
not sure if important or not?

Reply:
I'd say it's important to maintain backwards compatibility.

Reply:
But open to reasons for breaking it if they are also important :+1: